package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.12",
   "description": "Sparkbox implementation of drizzle-builder",
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=14.0.0"
```

Comment:
Do we also want to change the GitHub CI action to use 14?

Reply:
Ah, great call! All updated in 5ebc3295e6105b3c5bbbfbdbd982514f31b7656d.